### PR TITLE
DOCS-690, DOCS-707: Release updates

### LINKED ISSUE #690: [RELEASE] Operator v4.5.6 Doc Impacts
Operator release [4.5.6](https://github.com/minio/operator/releases/tag/v4.5.6) has changes that impact the docs:

- [ ] New options to expose services as a load balancer added to `kubectl minio tenant create` [PR #1374](https://github.com/minio/operator/pull/1374)
- [ ] New environment variables to support Console TLS [PR #1348](https://github.com/minio/operator/pull/1348)

### LINKED ISSUE #707: [RELEASE] Console release v0.22.5
Console release [v0.22.5](https://github.com/minio/console/releases/tag/v0.22.5) has changes the impact the docs:

- [ ] New Encryption tab for setting up KMS [PR #2377](https://github.com/minio/console/pull/2377)
- [x] Authorized users can perform `s3:Get*` actions from the Console [PR #2559](https://github.com/minio/console/pull/2559)

### PR DESCRIPTION
**Pull request by ravindk89**
Closes #690 

Closes #707 

Partially addresses scheduled removal of Console Audit Log and Tenant Prometheus deployments

Staged:

- http://192.241.195.202:9000/staging/k8s-fixups/operations/install-deploy-manage/deploy-minio-tenant.html#minio-tenant-audit-logging-settings
- http://192.241.195.202:9000/staging/k8s-fixups/operations/install-deploy-manage/deploy-minio-tenant.html#monitoring-settings
- http://192.241.195.202:9000/staging/k8s-fixups/reference/kubectl-minio-plugin/kubectl-minio-tenant-create.html
-

### CODE REVIEW

File: source/includes/common/common-k8s-deprecation-audit-prometheus.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+   You can instead use any webhook-capable database or logging service to capture :ref:`audit logs <minio-logging-publish-audit-logs>` from the Tenant.
+
+.. end-deprecate-audit-logs
+
+.. start-deprecate-prometheus
+
+.. important::
+
+   The Tenant Prometheus pod feature is scheduled for deprecation and removal in an upcoming release.
```
Comment (djwfyi) [inline source/includes/common/common-k8s-deprecation-audit-prometheus.rst:16]: This is probably fine in passive, but maybe:

> MinIO plans to deprecate and remove the Tenant Prometheus pod feature in an upcoming release.

or 

> We plan to ...

File: source/operations/install-deploy-manage/deploy-minio-tenant.rst [hunk 2]
```diff
@@ -644,9 +648,38 @@ Enabling SSE also creates :minio-git:`MinIO Key Encryption Service <kes>` pods i
...
+     - Description
+
+   * - Storage Class
+     - Select the storage class and requested capacity associated to the PVC generated to support Prometheus.
+
+   * - Storage Size
+     - Specify the amount of size of storage to make available for Prometheus.
+
+   * - :guilabel:`SecurityContext`
+     - The MinIO Operator assigns this Security Context for the Prometheus pod
```
Comment (djwfyi) [inline source/operations/install-deploy-manage/deploy-minio-tenant.rst:675]: Need a `.` at the end.

**Review by djwfyi — APPROVED** [review]
LGTM, mod nits

File: source/administration/console/managing-deployment.rst [hunk 3]
```diff
@@ -55,6 +55,12 @@ Select the :guilabel:`Start Logs` button to begin collecting logs using the sele
 Audit
 ~~~~~
 
+.. important::
+
+   The MinIO Tenant Console Audit Log feature is scheduled for deprecation and removal in an upcoming release.
+
+   You can instead use any webhook-capable database or logging service to capture :ref:`audit logs <minio-logging-publish-audit-logs>` from the Tenant.
```
[the remark this answers is not in the document]
Comment (ravindk89) [reply source/administration/console/managing-deployment.rst:62]: Done throughout.

**Pull request merged**